en/docs/security-guidelines/secure-engineering-guidelines/dynamic-analysis-with-owasp-zap.md

```diff
@@ -16,199 +16,199 @@ This document provides details of all necessary steps for configuring the OWASP
 ## OWASP ZAP Setup
 
 ### Increase JVM Heap Size for Running ZAP
-The heap size is defined in the *zap.sh* (for linux) and *zap.bat* (for Windows) files. Default value is set to `Xmx512m` (if available free memory is above 1,500 MB) and increase the value appropriately based on the memory availability of your system. (At-least 4GB is recommended)
+The heap size is defined in the *zap.sh* (for Linux) and *zap.bat* (for Windows) files. The default value is set to `Xmx512m` (if available free memory is above 1,500 MB) and increase the value appropriately based on the memory availability of your system. (At least 4GB is recommended)
 
-In addition to that, ZAP scan is a long running process. Therefore it is better to run ZAP in a cloud instance or in a dedicated server to avoid any interruptions. 
+In addition to that, the ZAP scan is a long-running process. Therefore, it is better to run ZAP in a cloud instance or in a dedicated server to avoid any interruptions. 
 
 
 ### Fine Tune ZAP Tool with Pre-Configured Policy 
-The ZAP tool should be fine tuned before running a scan for obtaining better results. For this, you can download the WSO2 policy file for ZAP[^2] which contains the settings to fine tune ZAP. 
+The ZAP tool should be fine-tuned before running a scan for obtaining better results. For this, you can download the WSO2 policy file for ZAP[^2] which contains the settings to fine-tune ZAP. 
 
 Go to **Analyze** &rarr; **Scan Policy** Manager in ZAP.
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-01.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-01.png)
 
-In the **Scan Policy Manager** window, click on **Import**.
+In the **Scan Policy Manager** window, click **Import**.
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-02.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-02.png)
 
 Browse and select the WSO2 Policy file you downloaded.
 
-![Placeholder]({{#base_path#}}/assets/images/secure-coding-guidelines/zap-03.png){ .post-image }
+![Placeholder](../../assets/images/secure-coding-guidelines/zap-03.png)
 
 Since the policy file is imported correctly, you can use this later when you run the spider and the scan. 
 
 
 ### Configuring ZAP Proxy to Trace Browser Traffic
-Rather than providing the URL of the WSO2 server and attacking the URL with ZAP, it is much more effective if we record the UI actions we do on the WSO2 server and let ZAP tool capture the traffic which can then be used for performing attacks. 
+Rather than providing the URL of the WSO2 server and attacking the URL with ZAP, it is much more effective if we record the UI actions we do on the WSO2 server and let the ZAP tool capture the traffic which can then be used for performing attacks. 
```

Comment:
```suggestion
Rather than providing the URL of the WSO2 server and attacking the URL with ZAP, it is much more effective if we record the UI actions we do on the WSO2 server and let the ZAP tool capture the traffic that can then be used for performing attacks. 
```